File: jdroid-gradle-googleplay-publisher-plugin/src/main/groovy/com/jdroid/android/googleplay/publisher/task/PublishMetadataTask.java
package com.jdroid.android.googleplay.publisher.task;

import com.jdroid.android.googleplay.publisher.App;
import com.jdroid.android.googleplay.publisher.GooglePlayPublisher;

public class PublishMetadataTask extends BaseTask {
	
	public PublishMetadataTask() {
		setDescription("Publish metadata (feature/promo graphics, High resolution icon, screenshots, title, short and full descriptions) on Google Play");
	}

	@Override
	protected void onExecute(App app) {
		GooglePlayPublisher.publishMetadata(app);
	}

}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PublishMetadataTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

open class PublishMetadataTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().publishMetadata(app)
    }

    init {
        description =
            "Publish metadata (feature/promo graphics, High resolution icon, screenshots, title, short and full descriptions) on Google Play"
    }
}
